src/index.js

```diff
@@ -168,38 +170,41 @@ class StickyTable extends React.Component {
       ['leftColumn', 'width', 'leftStickyColumnCount'],
       ['rightColumn', 'width', 'rightStickyColumnCount']
     ].forEach(([stickyKey, sizeKey, countPropKey]) => {
-      var insets = [0];
-      var count = props[countPropKey];
-      var netInset = 0;
+      var insets = [];
```

Comment:
seems odd to have `insets[0]` be `undefined` for count 1, and `insets[0]` be 0 for count 2+

Reply:
implying this should be `var insets = [0]`

Reply:
The first solution I had was to declare `var insets = props[countPropKey] > 1 ? [0] : [];` because the issue is that it sets to 0 all sides when any of the other stickyes is higher than 1

Reply:
I think my first comment in this thread is still relevant or the answer is unclear to me. I still don't understand why it should ever be empty. 

I actually wrote the code so that 0 would never be necessary here, so guessing insets should never have 0 in it, unless a cell has width 0. The first cell on a given side will always have an inset of 0 regardless of the status of the other sticky rows/columns

Reply:
seems like 0 should always be in insets, or never be in it